docs/src/components/frame/frame.module.css

```diff
@@ -9,23 +9,27 @@
   display: flex;
   flex-direction: column;
   min-height: 100vh;
+}
 
-  & .container {
-    display: flex;
-    flex-direction: row;
-    min-height: calc(100vh - var(--top-nav-height));
-  }
+.frame .container {
```

Comment:
Just for future reference - like when we get to TVA stuff, let's avoid nested selectors. That is a really bad pattern that by nature was created from preprocessors which we now know to be really bad.

Since this is the old docs, I'm not worried about it, but definitely want to get it out of your system to not do it in the future. 👍

Reply:
Gotcha.  That's good to know.  In this case I just flattened the SCSS syntax, since it was throwing errors.